src/restLevelClient/samples/generateSamples.ts

```diff
@@ -0,0 +1,23 @@
+export interface SampleGroup {
```

Comment:
Don't forget to add the license header in all new files

// Copyright (c) Microsoft Corporation.
// Licensed under the MIT License.